es6-destructuring/main.js

```diff
@@ -0,0 +1,54 @@
+const book1 = {
+  title: 'Goodnight Punpun',
+  author: 'Inio Asano',
+  libraryID: 3353,
+};
+
+const { title: title1, author: author1, libraryID: libraryID1 } = book1;
```

Comment:
Works, but you don't need `title1`, `author1`, `libraryID1`. You can just do something like the snippet below. You only need to give them a different variable name or `alias` if we already have a variable that matches the property name

`const { title, author, libraryID } = book1`